src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -248,6 +249,53 @@ def _executeConsumer(self, summary):
             self.logger.exception(msg)
             self.updateReportDict(summary, "error", msg)
 
+    def setDBSStatus(self, workflow):
+        """
+        The function to set the DBS status of outputs as VALID
+        :param workflow: a MSOutputTemplate object workflow
+        :return: the MSOutputTemplate object itself (with the necessary updates in place)
+        """
+        if not isinstance(workflow, MSOutputTemplate):
+            msg = "Unsupported type object '{}' for workflows! ".format(type(workflow))
+            msg += "It needs to be of type: MSOutputTemplate"
+            raise UnsupportedError(msg)
+
+        dbs3Writer = DBS3Writer(url=self.msConfig["dbsReadUrl"],
+                                writeUrl=self.msConfig["dbsWriteUrl"])
+
+        # if anything fail along the way, set it back to "pending"
+        dbsUpdateStatus = "done"
+        for dMap in workflow['OutputMap']:
+
+            if self.msConfig['enableDbsStatusChange']:
```

Comment:
I guess I would remove this option (and the dry-run mode). When we have it implemented, I believe there is no reason to disable this feature in MSOutput, right?

Reply:
I put it to make the switch from Unified to WMCore easy. So, once the PR is good to merge, we can merge it and put it to production with `enableDbsStatusChange`  `False` and simply it will not take any action but to print some logs. Once we're ready to switch from Unified to WMCore, we can simply change this parameter to True without touching the source code. Does it make sense?

Reply:
I think we could enable this DBS status change in MSOutput and keep both this service and Unified updating the DBS status for a day or two (I'm almost sure it's harmless and nothing fails). Then we have time to stop Unified.

At this stage, dry-run mode is just an over-complication of the code IMO.

Reply:
That makes sense to me. This option is removed in the latest commit.